src/core/lib/channel/http_server_filter.c

```diff
@@ -314,13 +314,17 @@ static grpc_error *init_call_elem(grpc_exec_ctx *exec_ctx,
   grpc_closure_init(&calld->hs_on_recv, hs_on_recv, elem);
   grpc_closure_init(&calld->hs_on_complete, hs_on_complete, elem);
   grpc_closure_init(&calld->hs_recv_message_ready, hs_recv_message_ready, elem);
+  gpr_slice_buffer_init(&calld->read_slice_buffer);
```

Comment:
Removed the other init: it was in error now.

The init needs to be here so that we can safely destroy the slice buffer always in the call op destroy path.